jobs/silk-cni/templates/cni-wrapper-plugin.conflist.erb

```diff
@@ -47,6 +47,11 @@
       'dns_servers' => p('dns_servers'),
       'host_tcp_services' => p('host_tcp_services'),
       'host_udp_services' => p('host_udp_services'),
+      'deny_networks' => {
+        'always' => p('deny_networks.always'),
+        'running' => p('deny_networks.running'),
+        'staging' => p('deny_networks.staging'),
+      },
```

Comment:
Can you build fast validation into the template to check if the operator passes a bad CIDR?

It would be preferable to fail fast here, rather than during a deploy.

Amelia and Aidan

Reply:
I'm not sure of an idiomatic way of doing this, but  I've had a go in https://github.com/cloudfoundry/silk-release/pull/23/commits/e8d57356cb2eeaffd00592edd7e79b98c5e86617 based on `jobs/silk-controller/templates/silk-controller.json.erb`